source/NVDAObjects/IAccessible/ia2TextMozilla.py

```diff
@@ -644,3 +645,27 @@ def _get_NVDAObjectAtStart(self):
 		if embedded:
 			return embedded
 		return obj
+
+	def _get_boundingRect(self):
+		rects = []
+		obj = self._startObj
+		ti = self._start
+		while obj:
+			if not obj.hasIrrelevantLocation:
+				try:
+					rects.append(ti.boundingRect)
+				except (NotImplementedError, LookupError):
+					pass
+			if obj == self._endObj:
+				# We're ad the end of the range.
+				break
+			try:
+				ti, obj = self._findNextContent(ti)
+			except LookupError:
+				# Can't move forward any further.
+				break
+			if obj == self._endObj:
+				ti = self._end

```

Comment:
Because that ti corresponds with the end of the range and its endOffset is the end of the current range. The earlier ti that is assigned by the call of _findNextContent is a textInfo for the object, but it might contain text that is out of the current range. I will add a comment to clarify this.